docs/modules/plugin/partials/auto-refresh-mojo-parameters.adoc

```diff
@@ -1,6 +1,8 @@
+[[interval]]
```

Comment:
I admit I have become more acostumed to the short notation `[#interval]`. https://docs.asciidoctor.org/asciidoc/latest/attributes/ids/#anchor 
I know it's nitpicking, but it allows more flexibility so we don't need to think about the limitations of the other notation. Would you mind using it? :pleading_face:

Reply:
ok no problem, I will change it.